src/test/java/org/tests/transaction/TestBatchModelFlush.java

```diff
@@ -75,17 +75,22 @@ public void multipleTopLevel_expect_singleFlush() {
     assertThat(sql).hasSize(9);
 
     // first saved to batch - (depth 100)
+    boolean hasSequence = isSqlServer();
     assertThat(sql.get(0)).contains("insert into mny_b");
-    assertThat(sql.get(1)).contains(" -- bind(BatchMultipleTop_0");
-    assertThat(sql.get(2)).contains(" -- bind(BatchMultipleTop_1");
+    if (!hasSequence) {
+      assertThat(sql.get(1)).contains(" -- bind(BatchMultipleTop_0");
+      assertThat(sql.get(2)).contains(" -- bind(BatchMultipleTop_1");
+    }
```

Comment:
I'm just wondering, how these tests could pass in my local test env. My connection settings are
```
ebean.mssql.idType=SEQUENCE
# Database has Collation: Latin1_General_100_BIN2
datasource.mssql.databaseUrl=jdbc:sqlserver://srv-01-docker02.foconis.local:1433;databaseName=ebean_unittest_bin;sendTimeAsDatetime=false
datasource.mssql.databaseDriver=com.microsoft.sqlserver.jdbc.SQLServerDriver
```
And I explicit enabeld SEQUENCE for SqlServer (by default, identity is used)
regarding your config file, SEQUENCE is commented out
https://github.com/ebean-orm/ebean/blob/master/src/test/resources/ebean.properties#L157

So sqlserver means *no sequencence* in your setup.
Nevertheless it would be better to use `spiEbeanServer().getServerConfig().getIdType() == IdType.SEQUENCE` instead of relying against a certain platform